src/devices/Ws28xx/BitmapImageNeo3.cs

```diff
@@ -19,20 +19,35 @@ internal class BitmapImageNeo3 : BitmapImage
         // data means we have to add 30 bytes of zero padding.
         private const int ResetDelayInBytes = 30;
 
+        public BitmapImageNeo3(int width, int height, ColorMode colorMode)
```

Comment:
Honestly I think you should just derive from `BitmapImageNeo3` and replace SetPixel with the new version

Reply:
and remove ColorMode as it feels like an implementation detail of the device in here

Reply:
OK